projects/project-1/assignment/MMiroir - P1.md

```diff
@@ -5,21 +5,44 @@ My results for assignment 2 vary significantly from the example chart provided.
-| **Asymm.** |OK      | OK      |X<sup>NS|X<sup>UNSAT| -         |X<sup>UNSAT| OK        |
-| **Ref.**   |OK      | OK      | OK     |     OK    |X<sup>UNSAT| -         |X<sup>UNSAT|
-| **Irref.** | OK     | OK      |X<sup>NS|     OK    |    OK     |X<sup>UNSAT| -         |
+|            | Funct. | iFunct.| Trans. | Symm.     | Asymm.    | Ref.      |Irref.     |
+|--------    |--------|--------|--------|-------    |--------   |-----      |--------   |
+| **Funct.** |-       | OK     |X<sup>NS| OK        | OK        | OK        | OK        |
+| **iFunct.**|OK      | -      |X<sup>NS| OK        | OK        | OK        | OK        |
+| **Trans.** |X<sup>NS|X<sup>NS| -      | OK        |X<sup>NS   | OK        |X<sup>NS   |
+| **Symm.**  |OK      | OK     | OK     | -         |X<sup>UNSAT| OK        | OK        |
+| **Asymm.** |OK      | OK     |X<sup>NS|X<sup>UNSAT| -         |X<sup>UNSAT| OK        |
+| **Ref.**   |OK      | OK     | OK     |     OK    |X<sup>UNSAT| -         |X<sup>UNSAT|
+| **Irref.** | OK     | OK     |X<sup>NS|     OK    |    OK     |X<sup>UNSAT| -         |
 
 **_provided by the assignment instructions_
 
-For example, to explain why `R` cannot be both asymmetric and symmetric, your explanation may take the form: 
-- Suppose `R` is both symmetric and asymmetric. Then by symmetry for any x and y, if x`R`y it follows that y`R`x. However, by asymmetry it also follows that it is not the case that y`R`x. Hence, `R` cannot be both symmetric and asymmetric.
-Similarly, to explain why `R` cannot be both transitive and inverse functional.
+### [Object Property Expression Axioms](https://www.w3.org/TR/owl2-direct-semantics/)
+
+ FunctionalObjectProperty( OPE ) 
+- ∀ x , y1 , y2 : ( x , y1 ) ∈ (OPE)<sup>OP</sup> and ( x , y2 ) ∈ (OPE)<sup>OP</sup> imply y1 = y2
+
+InverseFunctionalObjectProperty( OPE ) 
+-  ∀ x1 , x2 , y : ( x1 , y ) ∈ (OPE)<sup>OP</sup> and ( x2 , y ) ∈ (OPE)<sup>OP</sup> imply x1 = x2 
+
+TransitiveObjectProperty( OPE ) =
+- ∀ x , y , z : ( x , y ) ∈ (OPE)<sup>OP</sup> and ( y , z ) ∈ (OPE)<sup>OP</sup> imply ( x , z ) ∈ (OPE)<sup>OP</sup> 
+
+SymmetricObjectProperty( OPE ) 
+-  ∀ x , y : ( x , y ) ∈ (OPE)<sup>OP</sup> implies ( y , x ) ∈ (OPE)<sup>OP</sup> 
+
+ AsymmetricObjectProperty( OPE ) 
+-  ∀ x , y : ( x , y ) ∈ (OPE)<sup>OP</sup> implies ( y , x ) ∉ (OPE)<sup>OP</sup> 
+
+ ReflexiveObjectProperty( OPE ) 
+-  ∀ x : x ∈ ΔI implies ( x , x ) ∈ (OPE)<sup>OP</sup> 
+
+ IrreflexiveObjectProperty( OPE ) 
+- ∀ x : x ∈ ΔI implies ( x , x ) ∉ (OPE)<sup>OP</sup> 
+
+### Explanations
+
+_EXAMPLE GIVEN: To explain why `R` cannot be both asymmetric and symmetric, your explanation may take the form: Suppose `R` is both symmetric and asymmetric. Then by symmetry for any x and y, if x`R`y it follows that y`R`x. However, by asymmetry it also follows that it is not the case that y`R`x. Hence, `R` cannot be both symmetric and asymmetric. Similarly, to explain why `R` cannot be both transitive and inverse functional._
 
```

Comment:
If I understand correctly, for any functional property there can only be one bearer in relation to an instance. If xRy, then zRy is invalid because only (x) can bear this relation with (y). However, xRz is valid because an instance can be the bearer of a a functional property for multiple other instances.  So, when you say that by functionality if "xRy and xRz then y = z" I am not sure if that is necessarily true. I do understand the line of think about cardinality for functional object properties which implies that y must equal z (perhaps I am misinterpreting-- I could also be totally wrong!) I'm happy to expand more on this with concrete examples.

Reply:
Did some further research. I may be conflating functional with inverse functional.